engines/kingdom/constants.cpp

```diff
@@ -0,0 +1,5696 @@
+	"KLF026.lbm",
+	"KLF027.lbm",
+	"KLF028.lbm",
+	"KLF029.lbm",
+	"KLF030.lbm",
+	"KLF031.lbm",
+	"KLF032.lbm",
+	"KLF033.lbm",
+	"KLF034.lbm",
+	"KLF035.lbm",
+	"KLF036.lbm",
+	"KLF037.lbm",
+	"KLF038.lbm",
+	"KLF039.lbm",
+	"KLF040.lbm",
+	"KLF041.lbm",
+	"KLF042.lbm",
+	"KLF043.lbm",
+	"KLF044.lbm",
+	"KLF045.lbm",
+	"KLF046.lbm",
+	"KLF047.lbm",
+	"KLF048.lbm",
+	"KLF049.lbm",
+	"KLF050.lbm",
+	"KLF051.lbm",
+	"KLF052.lbm",
+	"KLF053.lbm",
+	"KLF054.lbm",
+	"KLF055.lbm",
+	"KLF056.lbm",
+	"KLF057.lbm",
+	"KLF058.lbm",
+	"KLF059.lbm",
+	"KLF060.lbm",
+	"KLF061.lbm",
+	"KLF062.lbm",
+	"KLF063.lbm",
+	"KLF064.lbm",
+	"KLF065.lbm",
+	"KLF066.lbm",
+	"KLF067.lbm",
+	"KLF068.lbm",
+	"KLF069.lbm",
+	"KLF070.lbm",
+	"KLF071.lbm",
+	"KLF072.lbm",
+	"KLF073.lbm",
+	"KLF074.lbm",
+	"KLF075.lbm",
+	"KLF076.lbm",
+	"KLF077.lbm",
+	"KLF078.lbm",
+	"KLF079.lbm",
+	"KLF080.lbm",
+	"KLF081.lbm",
+	"KLF082.lbm",
+	"KLF083.lbm",
+	"KLF084.lbm",
+	"KLF085.lbm",
+	"KLF086.lbm",
+	"KLF087.lbm",
+	"KLF088.lbm",
+	"KLF089.lbm",
+	"KLF090.lbm",
+	"KLF091.lbm",
+	"KLF092.lbm",
+	"KLF093.lbm",
+	"KLF094.lbm",
+	"KLF095.lbm",
+	"KLF096.lbm",
+	"KLF097.lbm",
+	"KLF098.lbm",
+	"KLF099.lbm",
+	"KLF100.lbm",
+	"KLF101.lbm",
+	"KLF102.lbm",
+	"KLF103.lbm",
+	"KLF104.lbm",
+	"KLF105.lbm",
+	"KLF106.lbm",
+	"KLF107.lbm",
+	"KLF108.lbm",
+	"KLF109.lbm",
+	"KLF110.lbm",
+	"KLF111.lbm",
+	"KLF112.lbm",
+	"KLF113.lbm",
+	"KLF114.lbm",
+	"KLF115.lbm",
+	"KLF116.lbm",
+	"KLF117.lbm",
+	"KLF118.lbm",
+	"KLF119.lbm",
+	"KLF120.lbm",
+	"KLF121.lbm",
+	"KLF122.lbm",
+	"KLF123.lbm",
+	"KLF124.lbm",
+	"KLF125.lbm",
+	"KLF126.lbm",
+	"KLF127.lbm",
+	"KLF128.lbm",
+	"KLF129.lbm",
+	"KLF130.lbm",
+	"KLF131.lbm",
+	"KLF132.lbm",
+	"KLF133.lbm",
+	"KLF134.lbm",
+	"KLF135.lbm",
+	"KLF136.lbm",
+	"KLF137.lbm",
+	"KLF138.lbm",
+	"KLF139.lbm",
+	"KLF140.lbm",
+	"KLF141.lbm",
+	"KLF142.lbm",
+	"KLF143.lbm",
+	"KLF144.lbm",
+	"KLF145.lbm",
+	"KLF146.lbm",
+	"KLF147.lbm",
+	"KLF148.lbm",
+	"KLF149.lbm",
+	"KLF150.lbm",
+	"KLF151.lbm",
+	"KLF152.lbm",
+	"KLF153.lbm",
+	"KLF154.lbm",
+	"KLF155.lbm",
+	"KLF156.lbm",
+	"KLF157.lbm",
+	"KLF158.lbm",
+	"KLF159.lbm",
+	"KLF160.lbm",
+	"KLF161.lbm",
+	"KLF162.lbm",
+	"KLF163.lbm",
+	"KLF164.lbm",
+	"KLF165.lbm",
+	"KLF166.lbm",
+	"KLF167.lbm",
+	"KLF168.lbm",
+	"KLF169.lbm",
+	"KLF170.lbm",
+	"KLF171.lbm",
+	"KLF172.lbm",
+	"KLF173.lbm",
+	"KLF174.lbm",
+	"KLF175.lbm",
+	"KLF176.lbm",
+	"KLF177.lbm",
+	"KLF178.lbm",
+	"KLF179.lbm",
+	"KLF180.lbm",
+	"KLF181.lbm",
+	"KLF182.lbm",
+	"KLF183.lbm",
+	"KLF184.lbm",
+	"KLF185.lbm",
+	"KLF186.lbm",
+	"KLF187.lbm",
+	"KLF188.lbm",
+	"KLF189.lbm",
+	"KLF190.lbm",
+	"KLF191.lbm",
+	"KLF192.lbm",
+	"KLF193.lbm",
+	"KLF194.lbm",
+	"KLF195.lbm",
+	"KLF196.lbm",
+	"KLF197.lbm",
+	"KLF198.lbm",
+	"KLF199.lbm",
+	"KLF200.lbm",
+	"KLF201.lbm",
+	"KLF202.lbm",
+	"KLF203.lbm",
+	"KLF204.lbm",
+	"KLF205.lbm",
+	"KLF206.lbm",
+	"KLF207.lbm",
+	"KLF208.lbm",
+	"KLF209.lbm",
+};
+
+const char *_movieNames[] = {
+	"",
```

Comment:
Same here, these are just sequentially numbered, it makes sense to calculate them at run time

Reply:
Agreed. I'll check the MAC version just in case, and I'll switch to this

Reply:
Arf, not the same in the mac version...